src/main/java/Duke.java

```diff
@@ -35,27 +39,54 @@ public static void main(String[] args) {
             // Create a string array to split user input into words
             String[] words = userInput.split(" ");
 
+            // Get description and schedule of task (exclude commands)
+            int descriptionPosition = userInput.indexOf(" ") + 1;
+            int timePosition = userInput.indexOf("/") + 4;
+            String description = userInput.substring(descriptionPosition);
+
+
+            switch(words[0]) {
             // Mark Task as Done
-            if(words[0].equals("done") || words[0].equals("Done")) {
+            case "done":
```

Comment:
Using a case instead of else if helps improve readability further down the road, good job!

Reply:
Have addressed this. Thanks for the feedback!